Permanent/Views/Cells/ArchiveTableViewCell/ArchiveScreenPendingArchiveDetailsTableViewCell.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ArchiveScreenPendingArchiveDetailsTableViewCell.swift
+//  Permanent
+//
+//  Created by Lucian Cerbu on 16.09.2021.
+//  Copyright © 2021 Victory Square Partners. All rights reserved.
+//
+
+import UIKit
+
+class ArchiveScreenPendingArchiveDetailsTableViewCell: UITableViewCell {
+    
+    @IBOutlet weak var archiveThumbnailImage: UIImageView!
+    @IBOutlet weak var archiveNameLabel: UILabel!
+    @IBOutlet weak var archiveAccessLabel: UILabel!
+    @IBOutlet var approveButton: RoundedButton!
+    @IBOutlet var denyButton: RoundedButton!
+    
+    var acceptButtonAction: ((ArchiveScreenPendingArchiveDetailsTableViewCell) -> Void)?
+    var declineButtonAction: ((ArchiveScreenPendingArchiveDetailsTableViewCell) -> Void)?
+    
+    var archiveData: ArchiveVOData?
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        
+        selectionStyle = .gray
+        
+        archiveNameLabel.font = Text.style16.font
+        archiveNameLabel.textColor = .darkBlue
+        
+        archiveAccessLabel.font = Text.style8.font
+        archiveAccessLabel.textColor = .darkBlue
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        contentView.backgroundColor = .white
```

Comment:
Selection of a row should be blocked in the corresponding tableView methods. shouldSelectRowAt: and shouldHighlightRowAt:

Reply:
Removed `setSelected` method from ArchiveScreenPendingArchiveDetailsTableViewCell.swift and added `tableView(_ tableView: UITableView, shouldHighlightRowAt indexPath: IndexPath) -> Bool ` in ArchivesViewController.swift

Where shouldSelectRowAt be used?